environment.yml

```diff
@@ -0,0 +1,34 @@
+name: geopandas-dev
+channels:
+    - conda-forge
+dependencies:
+    # required
+    - Cython>=0.16
+    - fiona>=1.7
+    - pandas>=0.23.4
+    - pyproj>=1.9.3
+    - shapely>=1.5
+
+    # geodatabase access
+    - psycopg2>=2.5.1
+    - SQLAlchemy>=0.8.3
+    
+    # geocoding
+    - geopy
+
+    # plotting
+    - descartes>=1.0
+    - matplotlib>=1.2.1
```

Comment:
I think we increased the minimum version for this at some point, the minimum we test is 2.0

Reply:
Thanks, missed that one!